docs/finalproject/external-projects.md

```diff
@@ -33,19 +29,14 @@ and to give our partners:
 
 More details can be found on the [Intro](./intro) and [Preparation](./prep) pages.
 
-## What does a Product Owner have to do?
+## We will need a Product Owner of your organisation on the team?
 
-To be a good Stakeholder/Product Owner you should
+Having an active Product Owner ensures the delivery is done faster and with the best solutions. Historically, one of the most common blockers to a successful project has been the quick communication between stakeholders and our trainees. Trainees may likely become blocked, spin wheels or go off on a tangent if they have a burning question and can't get it answered.
 
-- Be able to attend a weekly/bi-weekly meeting with our teams
-- Be in a position to advise the development of the project
-  - e.g. Should we prioritise Feature A or Feature B?
-  - e.g. We should remove Feature B because it's no longer needed.
-- Have realistic expectations of what our trainees can build in the time given to them
+In the ideal world, you would be able to answer questions within a 0.5-1 working days of the questions being asked by our trainees. If this might not be possible, let us know and we will select an Internal Product Manager who will be empowered to make decisions on your behalf.
```

Comment:
```suggestion
In an ideal world, you are able to answer trainee questions within one working day. If this might not be possible, let us know and we will select an Internal Product Manager who will be empowered to make decisions on your behalf.
```